transport-native-io_uring/src/test/java/io/netty/channel/uring/PollRemoveTest.java

```diff
@@ -64,6 +64,9 @@ public void initChannel(SocketChannel ch) { }
     @Test
     public void test() throws Exception {
         io_uring_test();
+
+        Thread.sleep(1000);
```

Comment:
A sleep in a test is always suspicious.

Reply:
I agree... that said this sleep has nothing to do with the PR itself ... I just added it as the test failed sometimes for me as the fd was not completely closed when we did rerun the test (was the case before as well)

Reply:
this test also failed for me as well..